### PR TITLE
Uses default sample rate and original audio without resampling

### PR DESCRIPTION
**Pull request by vanics**
Uses default sample rate and original audio without resampling sample for depgram live stream

This addresses the issue highlighted here:
https://github.com/livekit/agents/issues/202

### CODE REVIEW

File: livekit-plugins/livekit-plugins-deepgram/livekit/plugins/deepgram/stt.py [hunk 1]
```diff
@@ -279,9 +279,12 @@ async def send_task():
                 if isinstance(data, rtc.AudioFrame):
                     # TODO(theomonnom): The remix_and_resample method is low quality
                     # and should be replaced with a continuous resampling
-                    frame = data.remix_and_resample(
-                        self._sample_rate, self._num_channels
-                    )
+                    # frame = data.remix_and_resample(
```
Comment (vanics) [inline livekit-plugins/livekit-plugins-deepgram/livekit/plugins/deepgram/stt.py:282]: This is sort of an anti-pattern but wanted to leave here to check the intention of still reworking this.

File: livekit-plugins/livekit-plugins-deepgram/livekit/plugins/deepgram/stt.py [hunk 2]
```diff
@@ -149,7 +149,7 @@ def __init__(
         self,
         config: STTOptions,
         api_key: str,
-        sample_rate: int = 16000,
+        sample_rate: int = 44000,
```
Comment (keepingitneil) [inline livekit-plugins/livekit-plugins-deepgram/livekit/plugins/deepgram/stt.py:152]: 48000 is the common sample rate

**Comment by vanics** [comment]
@aviadr1 Why would it be 44100? I believe it is 48K. I just fixed it!

**Comment by aviadr1** [comment]
@vanics Oh, I am not sure what the original livekit sample rate is, but if it is not 48khz, then under that the customary sampling rate is 44100hz (and not 44000hz).

BTW is the livekit sampling rate ALWAYS 48000z?

File: livekit-plugins/livekit-plugins-deepgram/livekit/plugins/deepgram/stt.py (hunk 1 above)
Comment (keepingitneil) [reply livekit-plugins/livekit-plugins-deepgram/livekit/plugins/deepgram/stt.py:282]: hey @vanics can we actually keep the remix_and_resample method. If the sample rates match, it's no-op but we don't want it to completely break if users override the default.

Comment (keepingitneil) [reply livekit-plugins/livekit-plugins-deepgram/livekit/plugins/deepgram/stt.py:282]: Actually i'll merge and do a followup. Thanks for making this PR!

**Pull request merged**